resources/linux/redhat/atom.spec.in

```diff
@@ -20,6 +20,7 @@ Requires: lsb-core-noarch, libXss.so.1()(64bit) libsecret-1.so.0()(64bit), polki
 mkdir -p "%{buildroot}/<%= installDir %>/share/<%= appFileName %>/"
 cp -r "<%= appName %>"/* "%{buildroot}/<%= installDir %>/share/<%= appFileName %>/"
 mkdir -p "%{buildroot}/<%= installDir %>/bin/"
+%{?py3_shebang_fix: %py3_shebang_fix "../share/<%= appFileName %>/resources/app/apm/node_modules/npm/node_modules/node-gyp" "../share/<%= appFileName %>/resources/app/apm/node_modules/npm/node_modules/node-gyp/gyp/samples/samples"}  # Todo until node-gyp updates shebang to python3, fix Fedora33 build
```

Comment:
> `../share/<%= appFileName %>/resources [ . . . ]`

I don't think this path is quite right. I'm testing on Fedora and I get a "file not found" error.

Notes:

The spec steps seem to run by first `cd`ing into `~/rpmbuild/BUILD` which has (among other things) `Atom Dev/` in it with `resources/` directly under that.

The `%install` steps apparently copy various files from out of there to `~/rpmbuild/BUILDROOT/`, which ends up with `atom-dev-1.56.0.dev.1f0019953-0.1.fc33.x86_64/` under it, which has `usr/` under that. Like a mock filesystem ready for the package manager `dnf` to copy to the real filesystem at install time. (There are `usr/share/` and `usr/bin` files in there. Which I think matches where Atom actually installs to.)